ui/app/templates/components/server-agent-row.hbs

```diff
@@ -17,13 +17,20 @@
     />
   </span></td>
 <td data-test-server-is-leader>
-
-    <Hds::Badge
-      @text={{if this.agent.isLeader "True" "False"}}
-      @icon={{if this.agent.isLeader "check-circle" ""}}
-      @color={{if this.agent.isLeader "success" "neutral"}}
-      @size="large"
-    />
+  <Hds::Badge
+    @text={{if
+      this.agent.isLeader
+      (if
+        this.agent.system.shouldShowRegions
+        (concat "True" " (" this.agent.region ")")
+        "True"
+      )
+      "False"
+    }}
```

Comment:
Funky formatting, but:
1. If not a leader, just show "False"
2. If a leader, show "True"
3. If there are multiple regions at play, show region name in parentheses.